packages/components/src/AvatarGroup/AvatarGroup.module.css

```diff
@@ -0,0 +1,70 @@
+.AvatarGroup {
+  --avatar-sm: 1.25rem;
+  --avatar-md: 3rem;
+  --avatar-lg: 4.5rem;
```

Comment:
I think there's the same opportunity here to reduce this down to a single variable and update that value in the modifier classes